src/dataloader.ts

```diff
@@ -248,10 +248,10 @@ export abstract class Loader {
           else await unlink(errorPath).catch(() => {});
         }
         await prepareOutput(tempPath);
+        await prepareOutput(cachePath);
```

Comment:
This ensures that the cache folder exists before the data loader runs (so no `mkdir -p` needed).